thoth/si_bandit/version.py

```diff
@@ -18,4 +18,4 @@
 """This file carries the version and title of the si-bandit project."""
 
 __title__ = "thoth-si-bandit"
-__version__ = "0.1.0"
+__version__ = "0.1.1"
```

Comment:
This would be taken care by kebechet on releases. May be explicit change is not needed here.

Reply:
Oh ja right